src/Domain/DomainHelpers.hpp

```diff
@@ -72,3 +72,38 @@ std::vector<std::unique_ptr<CoordinateMapBase<Frame::Logical, TargetFrame, 3>>>
 wedge_coordinate_maps(double inner_radius, double outer_radius,
                       double inner_sphericity, double outer_sphericity,
                       bool use_equiangular_map) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// Generates the corners for a Domain which is made of one or more layers
+/// of Blocks fully enveloping an interior volume, e.g. Shell or Sphere. The
+/// `number_of_layers` specifies how many of these layers of blocks to have
+/// in the final domain.
+/// `include_central_block` is set to `true` in Sphere and `false` in Shell.
+/// The `central_block_corners` are used to seed the values for the corners
+/// of the surrounding blocks.
+std::vector<std::array<size_t, 8>> corners_for_radially_layered_domains(
+    size_t number_of_layers, bool include_central_block,
+    const std::array<size_t, 8>& central_block_corners = {
+        {1, 2, 3, 4, 5, 6, 7, 8}}) noexcept;
+
+/// \ingroup ComputationalDomainGroup
+/// Generates the corners for a BBH-like Domain which is made of one or more
+/// layers of Blocks surrounding two interior blocks. The
+/// `number_of_radial_layers` gives the number of layers that fully envelop
+/// each interior block with six blocks each. The `number_of_biradial_layers`
+/// gives the number of layers that fully cover both blocks at once, using ten
```

Comment:
I don't know what "fully cover both blocks at once" means. Are these layers surrounding the two "interior blocks"? Or are they overlapping them? If not, what does "cover" mean?

Reply:
These are the surrounding layers around the two "interior blocks", but I'm still thinking about a better wording